spec/features/user_can_delete_station_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../spec_helper'
+require_relative 'load_database'
+
+include LoadDatabase
+
+describe 'user deletes station' do
+  it 'can be deleted from show page' do
+    LoadDatabase.load
+    visit('/stations/2')
+    click_on('Delete')
+
+    page.has_xpath?('/stations')
```

Comment:
I used .has_xpath() because it was on a Capybara reference guide given to us by instructors and I found that before have_current_path(). Since it might complicate things, I will use have_current_path()